### PR TITLE
improve test coverage

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Comment by gpotter2** [comment]
@stephengroat Excuse me but what are you exactly trying to do ? Scapy has nothing to do with netiface ?! How is that going to improve the coverage ?! Why would we import and install new depedencies for useless tests ?!

**Comment by stephengroat** [comment]
@gpotter2 `iface` actually takes a `str` or a `list` (https://github.com/secdev/scapy/blob/216a941135ab903e8b2cda4ba2dd417c18c5c330/scapy/sendrecv.py#L595). I'm having an issue getting it working on some systems so I was wanted to add it to the tests for coverage. the `netifaces` is only used for testing, not a full dependency

The coverage that should be added is [this](https://github.com/secdev/scapy/blob/216a941135ab903e8b2cda4ba2dd417c18c5c330/scapy/sendrecv.py#L607) for `linux`, working on see how to do it for `windows` as well

**Comment by gpotter2** [comment]
@stephengroat My bad about the list :/
I don't really see why we would need a such test: scapy already detects the interfaces by itself.

Would your test check that scapy detects the interfaces well ?

**Comment by stephengroat** [comment]
@gpotter2 i don't see any decrease in code coverage in scapy's interface detection code, so that code still seems to be exercised. all this does is add test coverage

File: test/regression.uts [hunk 1]
```diff
@@ -841,11 +841,15 @@ arping(_+"/24")
...
 import os
+import netifaces
+from sys import platform
 def _send_or_sniff(pkt, timeout, flt, pid, fork, t_other=None):
     assert pid != -1
     if pid == 0:
         time.sleep(1)
-        (sendp if isinstance(pkt, (Ether, Dot3)) else send)(pkt)
+        return_packets = (sendp if isinstance(pkt, (Ether, Dot3)) else send)(pkt, return_packets=True)
+        if platform in ["linux", "linux2", "Darwin"]:
```
Comment (gpotter2) [inline test/regression.uts]: You can use the vars in scapy.consts

**Comment by gpotter2** [comment]
The more useless stuff you install, longer will be the builds...

**Comment by guedou** [comment]
I am closing this PR as no description is provided, and because builds are massively failing.

**Pull request closed without merging**